app/src/scenes/Items/index.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import axios, { AxiosResponse } from 'axios';
+import { handleApiError } from '../../services/util/errorHandler';
+import {
+  ItemPayload,
+  ItemStateProps,
+  SubCategory,
+  CategoryUrlParams,
+} from './interfaces';
+import { List } from 'antd';
+import { RouteComponentProps } from 'react-router';
+import styles from './styles.css';
+
+class Items extends React.Component<
+  RouteComponentProps<CategoryUrlParams>,
+  ItemStateProps
+> {
+  pageSize: number;
+  SubCategoryName: string;
```

Comment:
This should be in the state. (Because whenever we change the subcategory name this should be rerendered)